scripts/setup-ubuntu.sh

```diff
@@ -302,6 +302,9 @@ PACKAGES+=" libexpat1:i386"
 # Required by wine-stable
 PACKAGES+=" libfreetype-dev:i386"
 
+# Required by ibus
+PACKAGES+=" dconf-cli"
```

Comment:
Probably it should be built in host-building step or downloaded from ubuntu repos as it is done for audacity and aosp-libs. 
IMO build-time dependencies of single packages should be added to setup-ubuntu script only if there is no other choice.

Reply:
`dconf-cli` only takes up 90k of space, so I think it is ok to add it to image. In fact, a lot of GNOME software requires it to install configue files. Maybe it will be helpful in the future.

Reply:
:thumbsup: